Lib/defcon/objects/layoutEngine.py

```diff
@@ -38,15 +38,15 @@ def _layoutEngineOTLTablesRepresentationFactory(layoutEngine):
             addOpenTypeFeatures(feaPath, otf)
```

Comment:
Yeah. I don't think that was available when I added this functionality. Feel free to change it.